client/styles/components/_overlay.scss

```diff
@@ -42,3 +43,15 @@
   @include icon();
   padding: #{3 / $base-font-size}rem 0;
 }
+
+@media only screen and (max-width: 430px){
```

Comment:
Please don't just add random numbers for the breakpoints.  They should be variables used in Sass functions.